camp/mgetachew/mgetachewMiniProject/src/main/java/frc/robot/commands/moveMotor.java

```diff
@@ -0,0 +1,43 @@
+package frc.robot.commands;
+
+import java.util.function.BooleanSupplier;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.Subsystem;
+import frc.robot.subsystems.MotorTalon;
+
+public class moveMotor extends Command{
+    //globals here
+    MotorTalon motorTalonToMove;
+    BooleanSupplier rightBumperSupplier;
+    BooleanSupplier leftBumperSupplier;
+
+
+    public moveMotor(MotorTalon motorTalon, BooleanSupplier booleanXboxControllerSupplierLeft, BooleanSupplier booleanXboxControllerSupplierRight){
+        //require motor (or it wont work. Cant spin a motor without a motor)
+        addRequirements(motorTalon);
+        //construct the globals we set earlier
```

Comment:
change comment. these aren't globals, they are class variables. We would say something like: "save constructor inputs to class variables"